docs/overview/resources.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Other resources
+last_update:
+  date: 26 January 2024
+---
+
+If you need don't find the information that you need on this site, there are many other places that you can look:
+
+## Documentation
```

Comment:
Is there a reason to start with level-2 sections? (I'm not used to Markdown, so this may be a silly question)

Reply:
Level 1 is usually the page title.